### DISCUSSION TITLE
Visual Studio not highlihting GraphQL code from js?

### DISCUSSION THREAD

**Discussion by BJladika** [prisma/prisma#7512] [Q&A] [answered]
A bit a lame question, I think, but why my Visual Studio Code is not highlihting that gql part of code from the example https://github.com/prisma/prisma-examples/tree/latest/typescript/graphql-sdl-first
I mean, I've downloaded Apollo GraphQL and Prisma extensions and thought that it would be enough... (downloaded for tests also GraphQL extension, still no effect)

![image](https://user-images.githubusercontent.com/23732658/121020429-53a4b300-c7db-11eb-8513-e03670455343.png)

**Comment by BJladika** [comment, marked as answer]
Oh well, I guess adding `#graphql` tag like this can help (as described in GraphQL extension), but still a bit frustrating.
```
const typeDefs = `#graphql
etc etc...`
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
